auth_otp/services/auth_service.py

```
@@ -0,0 +1,437 @@
+                    'access': str(new_access),
+                    'token_type': 'Bearer',
+                    'expires_in': int(
+                        getattr(settings, 'SIMPLE_JWT', {}).get(
+                            'ACCESS_TOKEN_LIFETIME',
+                            timedelta(minutes=5)
+                        ).total_seconds()
+                    )
+                }
+                
+        except TokenError as e:
+            logger.error(f"Token refresh error: {e}")
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)